python/pyspark/ml/wrapper.py

```diff
@@ -264,7 +264,7 @@ def _to_java(self) -> "JavaObject":
         return self._java_obj
 
     @staticmethod
-    def _from_java(java_stage: "JavaObject") -> "JP":
+    def _from_java(java_stage: "JavaObject") -> "JP":  # type: ignore[type-var]
```

Comment:
@anilbey mind filing a JIRA and linking it into PR title? See also, https://spark.apache.org/contributing.html

Reply:
@HyukjinKwon here it is
https://issues.apache.org/jira/browse/SPARK-39811

Reply:
Let's link it to the PR title.